nixos/modules/misc/documentation.nix

```
@@ -413,6 +453,12 @@ in
         ];
     })
 
+    (mkIf cfg.nixpkgs.enable {
+      environment.systemPackages =
+        [ ]
+        ++ optionals cfg.doc.enable [
+          nixpkgs-help
+        ];
```

```suggestion
      environment.systemPackages = optional cfg.doc.enable nixpkgs-help;
```
No need to be this verbose.